app/controllers/webhooks_controller.rb

```diff
@@ -160,6 +160,9 @@ def handle_payment_intent_succeeded(
             email: payment_intent.purchaser.email
           }
         )
+      elsif payment_intent.campaign.present? && payment_intent.campaign.mega_gam?
+        payment_intent.successful = true
+        payment_intent.save!
```

Comment:
i realize we actually may want to send some email here but i'm not sure what we want to do here. should we be creating a new EmailManager?

Reply:
Yeah we probably should be sending an email

Reply:
It's also not super clear to me where the payment intent is set to successful in the other cases. I wonder if we can move all of the setting of successful out to the webhook, or move this in to a separate handler? Otherwise it looks like from here that the only case where the intent is set to successful is in the mega gam case, when that's not indeed true